src/main/java/fr/adrienbrault/idea/symfony2plugin/templating/util/TwigUtil.java

```diff
@@ -159,17 +161,17 @@
             shortcutName = String.format(
                 "%s::%s%s",
                 symfonyBundle.getName(),
-                templateFolderName,
-                templateName
+                underscore(templateFolderName),
```

Comment:
> With this change, we basically stop supporting the old < 4.0 syntax

Yes its just what i was think of, but i still see lot of projects having older versions. i think they can be easily supported as the method already supports multiple template names. I guess just adding the old naming to the end of the array will avoid this.

Reply:
That's clever! Gonna try that out.

Reply:
Changed it, tested it and it works. Tests are also green. Not sure if I need to add more test cases to also guard the old ones? How to do that?

Reply:
would be nice, but not needed for merging ;)